sql-schema/109.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE tmp_table LIKE config;
+ALTER IGNORE TABLE tmp_table ADD UNIQUE INDEX uniqueindex_configname (config_name);
```

Comment:
This is the name laravel picks when it creates indexes.  It isn't needed here, I can drop it if you like.